### ISSUE TITLE
TypeError when using Includable and $get

### ISSUE THREAD

**Issue by Bialogs** [RobinBuschmann/sequelize-typescript#925]
# Issue

## Versions

- sequelize: 6.5.0
- sequelize-typescript: 2.1.0
- typescript: 4.1.5

## Issue type

- [x] bug report
- [ ] feature request

## Actual behavior

`TypeError: options.include.push is not a function`

## Expected behavior

Query executes with include options.

## Steps to reproduce

Essentially the error shows up when using `$get` with include options and only passing a singular Model instead of an array of Models.

`user.$get('groups', {include: User});` Doesn't work
`user.$get('groups', {include: [User});` Works

Basically with the `$get` function you can't use `Includable` but must use `Includable[]` type

## Related code

Not to get you too involved with our current work in progress but you can see the change required here and all the additional context of the application: 

* Diff: https://github.com/mitre/heimdall2/pull/794/commits/6ba42cb59b6ef35d9f81a1deabc910ad96b11d42

**Comment by usmansbk** [comment]
Thanks. I had a hard time trying to figure out what I was doing wrong
